lib/timers.js

```diff
@@ -44,25 +44,25 @@ const TIMEOUT_MAX = 2147483647; // 2^31-1
 // list it is attached to.
 //
 //
-// ╔════ > Object Map
-// ║
-// ╠══
-// ║ refedLists: { '40': { }, '320': { etc } } (keys of millisecond duration)
-// ╚══          ┌─────────┘
-//              │
-// ╔══          │
-// ║ TimersList { _idleNext: { }, _idlePrev: (self), _timer: (TimerWrap) }
-// ║         ┌────────────────┘
-// ║    ╔══  │                              ^
-// ║    ║    { _idleNext: { },  _idlePrev: { }, _onTimeout: (callback) }
-// ║    ║      ┌───────────┘
-// ║    ║      │                                  ^
-// ║    ║      { _idleNext: { etc },  _idlePrev: { }, _onTimeout: (callback) }
-// ╠══  ╠══
-// ║    ║
-// ║    ╚════ >  Actual JavaScript timeouts
-// ║
-// ╚════ > Linked List
+// |---- > Object Map
+// |
+// |--
+// | refedLists: { '40': { }, '320': { etc } } (keys of millisecond duration)
+// |--          |---------|
+//              |
+// |--          |
+// | TimersList { _idleNext: { }, _idlePrev: (self), _timer: (TimerWrap) }
+// |         |----------------|
+// |    |--  |                              ^
+// |    |    { _idleNext: { },  _idlePrev: { }, _onTimeout: (callback) }
+// |    |      |-----------|
+// |    |      |                                  ^
+// |    |      { _idleNext: { etc },  _idlePrev: { }, _onTimeout: (callback) }
+// |--  |--
+// |    |
+// |    |---- >  Actual JavaScript timeouts
+// |
+// |---- > Linked List
```

Comment:
I think it would be okay to make an exception for this file and just add an `eslint-disable` line for the rule.

(It would be good to have this file as a test that we *can* use UTF-8 in sources, even if we prefer not to.)